Paintroid/src/main/java/org/catrobat/paintroid/dialog/SaveInformationDialog.kt

```diff
@@ -169,7 +169,7 @@ class SaveInformationDialog :
             when {
                 FileIO.isCatrobatImage -> presenter.showOraInformationDialog()
                 FileIO.compressFormat == Bitmap.CompressFormat.JPEG -> presenter.showJpgInformationDialog()
-                else -> presenter.showPngInformationDialog()
+                else -> presenter?.showPngInformationDialog()
```

Comment:
is this `?` really required, I think you made presenter lateinit var right 🤔? If `?` is required here, this file has other usages of presenter too, please update them too :)